lib/einhorn.rb

```diff
@@ -101,8 +101,18 @@ def self.default_state
     end
   end
 
+  def self.load_yaml(raw)
+    if RUBY_VERSION >= "2.6.0"
```

Comment:
Why not major bump the version and drop dead rubies ?

Reply:
It's a fair question.

Einhorn is low-level infrastructure and pretty stable in terms of functionality. I'd rather not drop older Rubies that still have a non-trivial amount of use if it's easy to test against them. 2.5 is my current limit. If older versions require an unreasonable amount of support, I will drop them.